chromium/debugging-rulesets.js

```diff
@@ -0,0 +1,20 @@
+/* global sendMessage */
+
+"use strict";
+
+window.onload = function() {
```

Comment:
Why is this triggered by `window.onload`? Why not just have it execute as it loads?

Reply:
I think it's a bit clearer, but I can remove this.